doc/application/application.rst

```diff
@@ -382,7 +382,8 @@ again.
 .. note:: The ``run`` target will use the QEMU binary available from the Zephyr
           SDK by default. To use an alternate version of QEMU, for example the
           version installed on your host or a custom version, set the
-          environment variable ``QEMU_BIN_PATH`` to the alternate path.
```

Comment:
thats defeats the purpose, if I have this in a file, then it will always be used. What we want is being able to switch between different versions of Qemu based on the session.